development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -149,36 +149,22 @@ manager.
 
 .. note:: If you are compiling Godot for production use, then you can
           make the final executable smaller and faster by adding the
-          SCons option ``target=release_debug``.
-
-          If you are compiling Godot with GCC, you can make the binary
-          even smaller and faster by adding the SCons option ``use_lto=yes``.
-          As link-time optimization is a memory-intensive process,
-          this will require about 7 GB of available RAM while compiling.
+          SCons option `production=yes`.
+          
+          If you wish, you can also enable features piecemeal with 
+          `optimize=speed` and/or `lto=full` which will enable
+          speed optimizations and link-time optimizations independently.
+          
+          In particular, you may not be able to build with LTO on with a weaker computer, 
+          as it can use quite a lot of RAM (up to 7GB). In this case, you may still be able to get
+          some benefits with weaker (but more memory-friendly) settings such as `lto=thin`, or
+          `lto=auto` which should choose a reasonable configuration for your machine.
 
 .. note:: If you want to use separate editor settings for your own Godot builds
           and official releases, you can enable
           :ref:`doc_data_paths_self_contained_mode` by creating a file called
           ``._sc_`` or ``_sc_`` in the ``bin/`` folder.
 
-Compiling a headless/server build
```

Comment:
Was this functionality removed?
Can't find anything in the original PR: https://github.com/godotengine/godot/pull/66242

Reply:
As I mentioned in the original post, I couldn't find how to do this, so I removed it so as not to give bad instructions. I'll probably revert this because another PR seems to fix the headless build instructions.